spec/catalog/api/minion/task_spec.rb

```diff
@@ -1,50 +1,59 @@
 require "catalog/minion"
 
-describe Catalog::Api::Minion::Order do
+describe Catalog::Api::Minion::Task do
   describe "#perform" do
-    let(:order) do
+    let(:task_minion) do
       described_class.new("localhost", "9092")
     end
 
     let(:message) { ManageIQ::Messaging::ReceivedMessage.new(nil, "Task.update", payload, nil, nil, nil) }
 
     let(:payload) { {"task_id" => "123"} }
     let(:payload_params) do
-      {:payload =>  message.payload, :message => message.message}
+      {:payload => message.payload, :message => message.message}
     end
     let(:config) do
       test_config = ::CatalogApiClient.configure
-      test_config.host = "catalog-api.catalog-ci.svc:8080"
+      test_config.host = "localhost:3000"
     end
 
     before do
       dummy_client = double("CatalogApiClient")
       allow(dummy_client).to receive(:configure).and_return(config)
-      stub_request(:post, "http://catalog-api.catalog-ci.svc:8080/internal/v1.0/notify/order_item/123")
+      stub_request(:post, "http://localhost:3000/internal/v1.0/notify/task/123")
     end
 
     context "when there is no error" do
-      it "posts a payload" do
-        order.perform(message)
-        expect(a_request(:post, "http://catalog-api.catalog-ci.svc:8080/internal/v1.0/notify/order_item/123").with(
+      it "posts a payload to the order item endpoint" do
+        task_minion.perform(message)
+        expect(a_request(:post, "http://localhost:3000/internal/v1.0/notify/task/123").with(
           :body    => payload_params,
           :headers => {
-            'X-Rh-Identity'=>'eyJlbnRpdGxlbWVudHMiOnsiaHlicmlkX2Nsb3VkIjp7ImlzX2VudGl0bGVkIjp0cnVlfX0sImlkZW50aXR5Ijp7ImFjY291bnRfbnVtYmVyIjoiY2F0YWxvZy1hcGktb3JkZXItbWluaW9uIn19'
+            'X-Rh-Identity'=>Base64.urlsafe_encode64({
+              :entitlements => {
+                :hybrid_cloud => {
```

Comment:
One small nit, this should just say `:ansible` now since we're no longer entitled to `:hybrid_cloud`.